packages/components/src/components/as-dropdown/as-dropdown.tsx

```diff
@@ -45,6 +45,15 @@ export class Dropdown {
    */
   @Prop() public showClearButton: boolean = false;
 
+  /**
+   * Function called when clicking outside of the dropdown.
+   * By default it closes the list.
+   *
+   * @type {function}
+   * @memberof Dropdown
+   */
+  @Prop() public onClickOutside: () => void = this.closeList.bind(this);
```

Comment:
```suggestion
  @Prop() 
  public onClickOutside: () => { 
      this._closeList();
  }
```

??

I think this works out of the box, there is no need for the public method `closeList` because if you want to change the default behaviour you just need to override this:

```js
dropdopn.onClickOutside = () => alert('foo');
```

Reply:
It's my fault for accepting the suggested change, but it was completely broken syntax-wise